docs/sources/flow/setup/start-agent.md

```diff
@@ -150,26 +150,102 @@ If you downloaded the standalone binary, you must run the agent from a terminal
 To start Grafana Agent on Linux, macOS, or FreeBSD, run the following command in a terminal window:
 
 ```shell
-AGENT_MODE=flow BINARY_PATH run CONFIG_FILE
+AGENT_MODE=flow BINARY_PATH run CONFIG_PATH
 ```
 
 Replace the following:
 
-* `BINARY_PATH`: The path to the Grafana Agent binary file
-* `CONFIG_FILE`: The path to the Grafana Agent configuration file.
+* `BINARY_PATH`: The path and Grafana Agent binary filename.
+* `CONFIG_PATH`: The path and Grafana Agent configuration filename.
 
 ### Start Grafana Agent on Windows
 
 To start Grafana Agent on Windows, run the following commands in a command prompt:
 
 ```cmd
 set AGENT_MODE=flow
-BINARY_PATH run CONFIG_FILE
+BINARY_PATH run CONFIG_PATH
 ```
 
 Replace the following:
 
-* `BINARY_PATH`: The path to the Grafana Agent binary file
-* `CONFIG_FILE`: The path to the Grafana Agent configuration file.
+* `BINARY_PATH`: The path and Grafana Agent binary filename.
+* `CONFIG_PATH`: The path and Grafana Agent configuration filename.
+
+### Set up Grafana Agent as a Linux systemd service
+
+You can set up and manage the standalone binary for Grafana Agent as a Linux systemd service.
+
+{{% admonition type="note" %}}
+These steps assume you have a default systemd and Grafana Agent configuration.
+{{% /admonition %}}
+
+1. To create a new user called `grafana-agent-flow` run the following command in a terminal window:
+
+   ```shell
+   sudo useradd --no-create-home --shell /bin/false grafana-agent-flow
+   ```
+
+1. Create a service file in `/etc/systemd/system` called `grafana-agent-flow.service` with the following contents:
+
+   ```shell
```

Comment:
This is not a shell snippet.

PrismJS supposedly supports the info string `systemd` for unit configuration file syntax highlighting.

Reply:
Updated in https://github.com/grafana/agent/pull/5455